editor/src/components/editor/canvas-toolbar.tsx

```diff
@@ -44,12 +49,140 @@ import { useToolbarMode } from './canvas-toolbar-states'
 import { when } from '../../utils/react-conditionals'
 import { StrategyIndicator } from '../canvas/controls/select-mode/strategy-indicator'
 import { toggleAbsolutePositioningCommands } from '../inspector/inspector-common'
+import { NO_OP } from '../../core/shared/utils'
+import type {
+  InsertableComponentFlatList,
+  InsertMenuItem,
+  InsertMenuItemValue,
+} from '../canvas/ui/floating-insert-menu'
+import {
+  CustomComponentOption,
+  useComponentSelectorStyles,
+  useGetInsertableComponents,
+} from '../canvas/ui/floating-insert-menu'
+import Select, { components, createFilter } from 'react-select'
+import WindowedSelect from 'react-windowed-select'
+import { InspectorInputEmotionStyle } from '../../uuiui/inputs/base-input'
+import { stopPropagation } from '../inspector/common/inspector-utils'
+import { useConvertTo } from './convert-callbacks'
+import { useWrapInDiv } from './wrap-in-callbacks'
 import { ElementsOutsideVisibleAreaIndicator } from './elements-outside-visible-area-indicator'
+import {
+  fragmentComponentInfo,
+  insertableComponentGroupFragment,
+} from '../shared/project-components'
 
 export const InsertMenuButtonTestId = 'insert-menu-button'
 export const InsertConditionalButtonTestId = 'insert-mode-conditional'
 export const CanvasToolbarId = 'canvas-toolbar'
 
+export interface ToolbarSearchListingProps {}
+
+export const CanvasToolbarSearchPortalId = 'canvas-toolbar-search-portal'
+
+export const ToolbarSearchListing = React.memo((props: ToolbarSearchListingProps) => {
```

Comment:
You can just do
```React.memo(() => {```
So there's no need for the empty `interface`

Reply:
Ah yes, at one point it wasn't empty, fixed in bee6ce5d3.